### PR TITLE
got add buttons working, fixed small cb select bug

### LINKED ISSUE #125: Recipe Card Plus Button
**Description**
 Add a `"click"` event listener to the `+` button on the recipe cards (inside of the function that creates the recipe cards!) that will display the Select Cookbook pop up and fetches the recipe data to store internally.

**Issue Points**: 1

### PR DESCRIPTION
**Pull request by grantDuntugan**
Resolves #125 

**Description**

Plus buttons work (I hope someone can test this)!

### CODE REVIEW

**Comment by grantDuntugan** [comment]
jsHint doesn't like my global variable solution help xd

File: source/scripts/app.js [hunk 1]
```diff
@@ -808,7 +810,7 @@ async function populateSelectCookbookOptions() {
 
   let cookbooks = await indexedDb.getAllCookbooks();
 
-  for (let i = 0; i < cookbooks.length; ++i) {
+  for (let i = 1; i < cookbooks.length; ++i) {
     let option = document.createElement("option");
     option.value = cookbooks[i].title;
     option.textContent = cookbooks[i].title;
```
Comment (minhdinh901) [inline source/scripts/app.js:816]: This implementation has a chance of populating the default cookbook twice, do this instead
```
for (let i = 0; i < cookbooks.length; ++i) {
  if (cookbooks[i].title !== DEFAULT_COOKBOOK_NAME) {
    let option = document.createElement("option");
    option.value = cookbooks[i].title;
    option.textContent = cookbooks[i].title;
    cookbookDropdown.append(option);
  }
}
```

File: source/scripts/spoonacular-interface.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ const MOCK_RECIPE_INFO = {
 };
 
 // Change this variable to true to make real API calls to Spoonacular
-let SPOONACULAR_ENABLED = true;
+let SPOONACULAR_ENABLED = false;
```
Comment (minhdinh901) [inline source/scripts/spoonacular-interface.js:53]: Change this back to `true`, so our deployed version has real recipe retrieval

File: source/scripts/app.js [hunk 3]
```diff
@@ -932,6 +949,18 @@ function bindExplorePageLearnMore() {
     let cardShadow = recipeCards[i].shadowRoot;
     let button = cardShadow.getElementById("recipe-info-button");
     button.addEventListener("click", redirectToRecipe);
+
+    /* jshint ignore:start */
+    let addButton = cardShadow.getElementById("recipe-card-action-button");
+    addButton.addEventListener("click", () => {
+      let recipeId = cardShadow.getElementById("recipe-id");
+      globalRecipeId = recipeId.textContent;
```
Comment (minhdinh901) [inline source/scripts/app.js:957]: You can avoid using `globalRecipeId` if you create a setter and getter inside the `notification-select-cookbook` custom element class to set and get a recipe ID

Comment (grantDuntugan) [reply source/scripts/app.js]: Added the setter and getter and removed global variable

**Review by minhdinh901 — APPROVED** [review]

**Review by adanestrada18 — APPROVED** [review]

**Pull request merged**